server/models/images.js

```diff
@@ -0,0 +1,12 @@
+import mongoose from "mongoose";
+
+const imageSchema= mongoose.Schema({
+    img:
+    {
+        data: Buffer,
```

Comment:
Is this schema necessary>?

Reply:
no, but I created it for the profile picture at that time, when I was implementing the compression techniques I am changing the name of it to profilePictureSchema